cve_bin_tool/checkers/openssh.py

```diff
@@ -27,27 +28,27 @@ def get_version(lines, filename):
             break  # The binary seems to contain many version strings and the
             # first one matches the binary in question
 
-    if filename in [
-        "scp",
-        "sftp",
-        "ssh",
-        "ssh-add",
-        "ssh-agent",
-        "ssh-argv0",
-        "ssh-copy-id",
-        "ssh-keygen",
-        "ssh-keyscan",
-        "slogin",
-    ]:
-        version_info["is_or_contains"] = "is"
-        version_info["modulename"] = "openssh-client"
-    elif filename in ["sshd"]:
-        version_info["is_or_contains"] = "is"
-        version_info["modulename"] = "openssh-server"
-
-    if "is_or_contains" in version_info:
```

Comment:
This statement didn't look like it did anything except always make the modulename openssl if either openssh-server or openssh-client had been set to the modulename, maybe I'm wrong though. I decided to just return as soon as we've identified.